### DISCUSSION TITLE
Enhancement: ReadPlayerID parameter for ChangeAnim

### DISCUSSION THREAD

**Discussion by LasombraDemon** [ikemen-engine/Ikemen-GO#891] [Feature Request] [answered]
Hey there! 

I know we have ChangeAnim2 to refer someone to its own animations during a customstate.

But it could be useful to have a way to put a player into anyone else's animation frames. SelfState has the ReadPlayerID parameter. 

ChangeAnim could use something like that, too.

**Comment by K4thos** [comment, marked as answer]
https://github.com/ikemen-engine/Ikemen-GO/wiki/State-controllers-(changed)#readplayerid
